### DISCUSSION TITLE
Set page title

### DISCUSSION THREAD

**Discussion by brentbaum** [idom-team/idom#400] [Question] [answered]
Is there a way to either set the page title from within idom or pass my own index.html? The latter would be a more extensible solution IMO.

**Comment by rmorshea** [comment, marked as answer]
I think this issue captures the request more generally: https://github.com/idom-team/idom/issues/253

I'm not completely sure what I want to do here though. While this level of customization seems completely reasonable, I'll have to think about how to draw the line between slight customizations, and changes that ought to require you to implement your own client. The latter wouldn't be a huge problem though since you'd be able to use the `idom-client-react` JS library.
